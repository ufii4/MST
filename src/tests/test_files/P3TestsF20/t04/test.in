n 4
i 0;1;10
i 0;2;6
i 0;3;5
i 2;3;4
i 1;3;15
edge_count
mst
exit
