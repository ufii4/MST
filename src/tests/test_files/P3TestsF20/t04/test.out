success
success
success
success
success
success
edge count is 5
mst 19.00
